src/components/partner/partner.repository.ts

```diff
@@ -80,10 +84,17 @@ export class PartnerRepository extends DtoRepository(Partner) {
     return result.dto;
   }
 
-  protected hydrate() {
+  protected hydrate(session: Session) {
     return (query: Query) =>
       query
-        .apply(matchProps())
+        .optionalMatch([
+          node('project', 'Project'),
+          relation('out', '', 'partnership'),
+          node('', 'Partnership'),
+          relation('out', '', 'partner'),
+          node('node'),
+        ])
```

Comment:
I'm not sure the changes from 7ee2fd2 make sense for partners & organizations. There can be multiple projects matched for these `nodes` and therefore the `scope` would just be the from the first project found. I think it's going to take some extra work on these two objects.

I think the correct solution is to merge the scope from all projects right? As being X on any project grants you rights to org/partner as that role.